apache/incubator-kyuubi
# [BUILD] Fix source release build

### _Why are the changes needed?_

Support build, run unit tests, and create binary tarball from source release.

### _How was this patch tested?_


- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

The verify steps:

1. create a source release tarball 
```
RELEASE_VERSION=1.4.0-SNAPSHOT build/release/create-package.sh source
```
2. untar tarball
```
cd work/release
tar -xzf kyuubi-1.4.0-SNAPSHOT-bin.tgz
cd kyuubi-1.4.0-SNAPSHOT-bin
```
3. compile and run unit test
```
build/mvn clean install
```
4. create binary release tarball
```
build/dist --tgz --spark-provided -Pkyuubi-extension-spark-3-1
```
